app/views/editorial/submissions/_section_home.html.haml

```diff
@@ -0,0 +1 @@
+= f.input :short_summary
```

Comment:
Is short_summary an attribute of Node? If so, should the form entry be in `new.html.haml` rather than the subclass specific partials?